### DISCUSSION TITLE
Error BlInitializeLibary failed 0xc0000001, when running on a real computer

### DISCUSSION THREAD

**Discussion by TiM-SyStEm** [CosmosOS/Cosmos#2057] [Q&A]
Hello, I tried to run my OS on a real computer, and I have this error: BlInitializeLibary failed 0xc0000001, does anyone know why it appears and how to fix it?

**Comment by valentinbreiz** [comment]
Hello @TiM-SyStEm, is your BIOS in UEFI or Legacy mode?
